pages/dashboard.vue

```diff
@@ -44,8 +44,8 @@
                 online videos, and downloadable quick guides.
               </p>
               <v-row no-gutters class="align-bottom pt-7">
-                <span class="primary--text font-weight-bold">Go to the Registry Application Help Page</span>
-                <v-icon color="primary" small right>mdi-open-in-new</v-icon>
+                <span class="primary2--text font-weight-bold">Go to the Registry Application Help Page</span>
```

Comment:
I'm not sure if this will work.

Vuetify may construct these text colour classnames (eg, primary--text) before your new colour name is declared in nuxt.config.js.

Reply:
It's been working, so I guess it's fine?

Reply:
Cool. I just learned something :)

Reply:
I don't have permission to resolve this conversation, but it's resolved.